internal/applicationsnapshot/report.go

```diff
@@ -235,6 +238,40 @@ func condensedMsg(results []evaluator.Result) map[string][]string {
 	return shortNames
 }
 
+func generateMarkdownSummary(r *Report) ([]byte, error) {
+	var markdownBuffer bytes.Buffer
+	markdownBuffer.WriteString("| Field     | Value |Status|\n")
+	markdownBuffer.WriteString("|-----------|-------|-------|\n")
+
+	var totalViolations, totalWarnings, totalSuccesses int
+	pr := r.toSummary()
+	for _, component := range pr.Components {
+		totalViolations += component.TotalViolations
+		totalWarnings += component.TotalWarnings
+		totalSuccesses += component.TotalSuccesses
+	}
+
+	writeIcon := func(condition bool) string {
+		if condition {
+			return ":white_check_mark:"
+		}
+		return ":x:"
+	}
+
+	writeMarkdownField(&markdownBuffer, "Time", r.created.UTC().Format("2006-01-02 15:04:05"), "")
+	writeMarkdownField(&markdownBuffer, "Name", " ", "")
+	writeMarkdownField(&markdownBuffer, "Successes", totalSuccesses, writeIcon(totalSuccesses >= 1 && pr.Success))
+	writeMarkdownField(&markdownBuffer, "Failures", totalViolations, writeIcon(totalViolations == 0))
+	writeMarkdownField(&markdownBuffer, "Warnings", totalWarnings, writeIcon(totalWarnings == 0))
+	writeMarkdownField(&markdownBuffer, "Result", r.Success, writeIcon(r.Success))
```

Comment:
Perhaps we don't need to print true/false
```suggestion
	writeMarkdownField(&markdownBuffer, "Result", "", writeIcon(r.Success))
```